packages/waku/src/lib/plugins/vite-plugin-deploy-vercel.ts

```diff
@@ -140,7 +140,7 @@ export function deployVercelPlugin(opts: {
           );
         }
         const vcConfigJson = {
-          runtime: 'nodejs20.x',
+          runtime: 'nodejs22.x',
```

Comment:
Let's see if this works.

```suggestion
```

Reply:
Thanks for the experiment. Yeah, I think I followed the build-output-api docs.

Alright. Let's accept your original suggestion.